package.json

```diff
@@ -35,6 +36,7 @@
 		"idle"
 	],
 	"dependencies": {
+		"@types/node": "^11.9.5",
```

Comment:
Should this really be a dependency? I think it's expected that Node.js project import this themselves.

Reply:
Seems to be the case: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
That only talks about npm packages, not built-in Node.js modules. I've been putting it in devDependencies in other packages, and it's fine: https://github.com/sindresorhus/hook-std/blob/f429639e9d60ae3259a17cd4aefaf62a9e4f15c1/package.json#L40